gravitee-apim-portal-webui-next/src/services/portal.service.ts

```diff
@@ -44,4 +45,13 @@ export class PortalService {
   public getApiInformations(apiId: string): Observable<ApiInformation[]> {
     return this.http.get<ApiInformation[]>(`${this.configService.baseURL}/apis/${apiId}/informations`);
   }
+
+  public getPortalHomepage(includeContent: boolean = true): Observable<PortalPage[]> {
+    const params = new URLSearchParams();
+    params.set('type', 'HOMEPAGE');
+    if (includeContent) {
+      params.set('expands', 'CONTENT');
+    }
+    return this.http.get<PortalPage[]>(`${this.configService.baseURL}/portal-pages?${params.toString()}`);
+  }
```

Comment:
```suggestion
  public getPortalHomepages(): Observable<PortalPage[]> {
    return this.http.get<PortalPage[]>(`${this.configService.baseURL}/portal-pages}`, {
       params: {
          type: 'HOMEPAGE',
          expands: 'CONTENT',
       }
    });  }
```

- Rename the method to `getPortalHomepages` since it returns a list
- Add params directly in the call since we will need the type and the content every time when getting the homepage